### ISSUE TITLE
Post Featured Image Block: Consider adding a focal point picker 

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#38899]
This feedback came up as part of the [FSE Outreach Program's All Things Media exploration](https://make.wordpress.org/test/2022/02/02/fse-program-exploration-all-things-media/#comment-2325): 

>Around 13:30 I created my featured image. When I went into the post image and started changing the size, I wondered why I didn’t have the ability to create a focus or essential part of the featured image. One of my frustrations with featured images in the past has been having images poorly cropped. Maybe a future feature would be that we can identify a focus when we upload a feature image, to better display those images cropped? Especially if we have scale > cover as an option.

To be clear, I'm talking about this option that's built into the Cover block:

<img width="276" alt="Screen Shot 2022-02-17 at 3 53 11 PM" src="https://user-images.githubusercontent.com/26996883/154591253-6bd988ac-fc79-480a-a4f2-0ca5e87670ed.png">

Here's an example from my personal site with TT2 where the featured image is cut off for example:

<img width="1160" alt="Screen Shot 2022-02-17 at 3 54 24 PM" src="https://user-images.githubusercontent.com/26996883/154591412-913b2dc1-20f2-4625-a3b7-1385ec397742.png">

The biggest question mark in my mind is how setting the focal point wouldn't necessarily solve the odd cropping issues as whatever the focal point was set to would apply to all featured images. Either way, wanted to pass along for more thoughts as it was an interesting take :)

**Comment by jasmussen** [comment]
It would indeed be nice to find a focal picker component that is more compressed in its resting state than what we have now, so that we can use it in more places. Without further changes, in the case of the featured image block you'd have to just decide the focal point without any context, but that could still be valuable.

Interestingly, many smartphones today [store metadata in images as to where faces are present](https://iptc.org/std/photometadata/specification/IPTC-PhotoMetadata#image-region). It could be interesting if that metadata could automatically be read and applied 🤔

**Comment by mtias** [comment]
We'd need to store the coordinates in a custom field to support this.

**Comment by Jastos** [comment]
I have added this feature to my theme to avoid images from having heads chopped off when the post featured image block is in Cover mode. It works well, but I did have to use custom fields.  It took me a while to figure out why it wasn't working with custom post types - because you have to turn on custom field support for custom post types.

**Comment by Marc-pi** [comment]
yes, that's a big (daily) need, as explained by @annezazu 
write a post, add your featured image, but in your post the image is croped on the wrong area because there is no focus selector...
i don't know if we have to merge the image and the cover block, for sure in DB that's another story -;-)
poke @richtabor

**Comment by dhanson-wp** [comment]
Agreed with @Marc-pi that this is a pretty regular need in the post editing experience. Clients continually ask how their images should be formatted _before_ uploading them to fit within the designs of query loops and post templates. You should be able to forgo specific image cropping in favor of the focal point selector.

**Comment by Mamaduka** [comment]
Related #20321.

**Comment by Mamaduka** [comment]
Closing as duplicate of #20321.
